google/cloud/bigquery_storage_v1/types.py

```diff
@@ -14,38 +14,67 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
 from __future__ import absolute_import
+
+import collections
+import inspect
 import sys
 
-from google.api_core.protobuf_helpers import get_messages
+import proto
 
-from google.cloud.bigquery_storage_v1.proto import arrow_pb2
-from google.cloud.bigquery_storage_v1.proto import avro_pb2
-from google.cloud.bigquery_storage_v1.proto import storage_pb2
-from google.cloud.bigquery_storage_v1.proto import stream_pb2
+from google.cloud.bigquery.storage_v1.types import arrow
+from google.cloud.bigquery.storage_v1.types import avro
+from google.cloud.bigquery.storage_v1.types import storage
+from google.cloud.bigquery.storage_v1.types import stream
+
+from google.protobuf import message as protobuf_message
 from google.protobuf import timestamp_pb2
 
 
+# The current api core helper does not find new proto messages of type proto.Message,
```

Comment:
Is this a manual change or autogenned?

Reply:
I recall you mentioned we need to use the proto messages directly because the proto-plus wrappers are too slow. Could you add some comments and/or docstrings describing this?

Reply:
Actually, this fix is to make sure that the generated proto classes are actually hoisted to the `types` module, it's not about performance.

The thing is that the helper from core currently only hoists subclasses of `google.protobuf.message.Message`, but skips subclasses of `proto.Message` (the latter is the type of the new messages generated by microgenerator). It's the same fix that had to be used in PubSub's `types` module.

The PubSub throughput performance fix (circumventing proto-plus wrappers) that you mentioned had to be used in a few key spots in PubSub code, because handling _many_ small-ish messages actually used too much CPU, which dominated the I/O time, and circumventing the wrappers helped there.